google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/GeneratedCreateTableStatementsTests.java

```diff
@@ -282,8 +282,8 @@ public void testDropTables() throws SQLException {
           .startsWith(
               "START BATCH DDL",
               "drop index if exists name_index",
-              "drop table `Employee`",
-              "drop table `Employee_Sequence`",
+              "drop table Employee",
+              "drop table Employee_Sequence",
```

Comment:
Why does this change remove identifier quoting? I think that this might break existing customers who use table names that contain reserved keywords.